apps/src/templates/InputPrompt.jsx

```diff
@@ -44,12 +47,22 @@ export default class InputPrompt extends Component {
     this.refs.answer.focus();
   }
 
+  handleChange = e => {
+    this.setState({value: e.target.value});
+  };
+
   render() {
     return (
       <form style={styles.form} onSubmit={this.handleSubmit}>
         <div style={styles.wrapper}>
-          <p style={styles.question}>{this.props.question}</p>
-          <input ref="answer" type="text" style={styles.input} />
+          <label style={styles.question}>{this.props.question}</label>
+          <input
+            ref="answer"
+            value={this.state.value}
+            onChange={this.handleChange}
```

Comment:
nit: you could inline the function here since it's one line: `onChange={e => this.setState({value: e.target.value})}`